sql/core/src/test/scala/org/apache/spark/sql/test/SQLTestUtils.scala

```diff
@@ -363,6 +363,18 @@ private[sql] trait SQLTestUtilsBase
     }
   }
 
+  /**
+   * Drops namespace `namespace` after calling `f`.
+   * Note that, do not switch current catalog in `f`. It may remove the wrong namespace.
```

Comment:
`Note that, if you switch current catalog/namespace in `f`, you should switch it back manually.`